test/framework/deployment_helpers.go

```diff
@@ -94,31 +94,75 @@ func DescribeFailedDeployment(input WaitForDeploymentsAvailableInput, deployment
 	return b.String()
 }
 
-// WatchDeploymentLogsInput is the input for WatchDeploymentLogs.
-type WatchDeploymentLogsInput struct {
+// WatchDeploymentLogsBySelectorInput is the input for WatchDeploymentLogsByLabelSelector.
+type WatchDeploymentLogsBySelectorInput struct {
+	GetLister GetLister
+	ClientSet *kubernetes.Clientset
+	Labels    map[string]string
+	LogPath   string
+}
+
+// WatchDeploymentLogsByDeploymentInput is the input for WatchDeploymentLogsByDeployment.
+type WatchDeploymentLogsByDeploymentInput struct {
+	GetLister  GetLister
+	ClientSet  *kubernetes.Clientset
+	Deployment *appsv1.Deployment
+	LogPath    string
+}
+
+// watchDeploymentLogsInput is the input for watchDeploymentLogs.
+type watchDeploymentLogsInput struct {
 	GetLister  GetLister
 	ClientSet  *kubernetes.Clientset
 	Deployment *appsv1.Deployment
```

Comment:
```suggestion
// watchPodLogsInput is the input for watchPodLogs.
type watchPodLogsInput struct {
	GetLister  GetLister
	ClientSet  *kubernetes.Clientset
	Namespace string
```

This now applicable for pods ;-)

Reply:
I have made the changes that you have asked, kindly review.